packages/fxa-payments-server/src/components/AlertBar/index.tsx

```diff
@@ -3,20 +3,65 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-
+import { ReactComponent as CloseIcon } from 'fxa-react/images/close.svg';
+import { Localized } from '@fluent/react';
 import Portal from 'fxa-react/components/Portal';
 import './index.scss';
 
 type AlertBarProps = {
+  actionButton?: any;
+  checked?: boolean;
   children: any;
   className?: string;
+  dataTestId: string;
+  descId: string;
+  elems?: boolean;
+  headerId: string;
+  localizedId: string;
+  onClick?: Function;
 };
 
-export const AlertBar = ({ children, className = 'alert' }: AlertBarProps) => {
+export const AlertBar = ({
+  actionButton,
+  checked,
+  children,
+  className = 'alert',
+  dataTestId,
+  descId,
+  elems,
+  headerId,
+  localizedId,
+  onClick,
+}: AlertBarProps) => {
   return (
-    <Portal id="top-bar">
-      <div data-testid="alert-container" className={className}>
-        {children}
+    <Portal id="top-bar" {...{ headerId, descId }}>
+      <div id={headerId} data-testid="alert-container" className={className}>
+        <Localized id={localizedId} elems={elems ? { div: actionButton } : undefined}>
+          <span
+            id={descId}
+            data-testid={dataTestId}
+            className={checked ? "checked" : undefined}
+          >
+            {children}
+          </span>
+        </Localized>
+
+        {onClick && <Localized id="close-aria">
+            <span
+              data-testid="clear-success-alert"
+              className="close"
+              aria-label="Close modal"
```

Comment:
Since this is a `span` (non-descriptive element), a `role` attribute will have to be added for the `aria-label` to work as intended.